### PR TITLE
feat: Hide unknown medium text

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves [CX-2413](https://artsyproduct.atlassian.net/browse/CX-2413)

Before
image

After
image

### CODE REVIEW

File: src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx [hunk 1]
```diff
@@ -616,15 +622,21 @@ const renderSmallCollapse = (props, user, mediator, filtersAtDefault) => {
         <Row mb={2}>
           <Col xs={4}>
             <Text variant="xs" fontWeight="bold">
-              Artwork Info
+              {categoryText !== "Unknown" && mediumText !== "Unknown"
+                ? "Artwork Info"
+                : "Artwork Dimension"}
```
Comment (rajsam003) [inline src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx:627]: Added the condition for the title as well. As if `categoryText` and `mediumText` both are unknown we will have only `dimension_text`

File: src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx [hunk 2]
```diff
@@ -542,12 +544,16 @@ const renderLargeCollapse = (props, user, mediator, filtersAtDefault) => {
         <Row>
           <Col sm={2}>
             <Text variant="xs" fontWeight="bold">
-              Artwork Info
+              {categoryText !== "Unknown"
+                ? "Artwork Info"
+                : "Artwork Dimension"}
```
Comment (rajsam003) [inline src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx:549]: Added the condition for the title as well. As if `categoryText` is unknown we will have only dimension_text

**Review by MrSltun — APPROVED** [review]
LGTM! Great work Sam! 👏 👏

File: src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx (hunk 2 above)
Comment (MounirDhahri) [reply src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx:549]: I think this should be `dimensions` instead of `dimension`

**Review by MounirDhahri — COMMENTED** [review]
Would you mind adding a test for this to make sure we don't mistakenly bring this back in the future

**Pull request merged**